src/main/java/com/aws/greengrass/deployment/DeviceConfiguration.java

```diff
@@ -275,7 +275,7 @@ void persistInitialLaunchParams(KernelAlternatives kernelAlts) {
             String jvmOptions = ManagementFactory.getRuntimeMXBean().getInputArguments().stream().sorted()
                     .filter(s -> !s.startsWith(JVM_OPTION_ROOT_PATH)).collect(Collectors.joining(" "));
             kernel.getConfig().lookup(SERVICES_NAMESPACE_TOPIC, getNucleusComponentName(), CONFIGURATION_CONFIG_KEY,
-                    DEVICE_PARAM_JVM_OPTIONS).dflt(jvmOptions);
+                    DEVICE_PARAM_JVM_OPTIONS).withNewerValue(DEFAULT_VALUE_TIMESTAMP + 1, jvmOptions);
```

Comment:
`DEFAULT_VALUE_TIMESTAMP` is 1, right? so `DEFAULT_VALUE_TIMESTAMP + 1` is 2? Why use 2?

Reply:
I could also use the current timestamp.
But thinking of the scenario of adding thing to existing thinggroup (ts2) with an active deployment, if the deployment changes jvmOptions (ts1), ts1<ts2, should the desired behavior be thing picking up the updated jvmOptions from the deployment? If yes, we cannot use the current timestamp.

Reply:
Also I just thought of another case, if nucleus reads in a config.yaml file (with file last modified timestamp), the file can also override jvmOptions if needed.